git-multimail/git_multimail.py

```diff
@@ -1104,6 +1107,7 @@ def _compute_values(self):
             oneline = oneline[:max_subject_length - 6] + ' [...]'
 
         values['rev'] = self.rev.sha1
+        values['parents'] = ' '.join(self.parents)
```

Comment:
Ideally, there should be a test showing this new template value in action.

Reply:
It'd be nice if you could update this PR with one more test. There's a `t/test_templates.py` used by `t/email-content.t` that should help (I have to admit I don't remember much about how it works ;-), but it should be relatively self-explanatory).

Reply:
PR updated with requested changes.